### PR TITLE
[Flink 31170] [docs]The spelling error of the document word causes sq…

### PR DESCRIPTION
**Pull request by BoYiZhang**
## What is the purpose of the change

  - *Fix spelling errors in the create table statement*

## Brief change log

  - *The create table statement provided by the document is incorrect, and an error is reported during execution*

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (no)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (no)
  - The serializers: (yes / no / don't know)
  - The runtime per-record code paths (performance sensitive): (no )
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn, ZooKeeper: (no)
  - The S3 file system connector: (no)

## Documentation

  - Does this pull request introduce a new feature? ( no)
  - If yes, how is the feature documented? (not documented)

### CODE REVIEW

**Comment by BoYiZhang** [comment]
This is my first PR, please help me find out where it doesn't meet the regulations ...
@reswqa @healchow

**Review by MartijnVisser — CHANGES_REQUESTED** [review]
@BoYiZhang Thanks for the fix, but then we should also remove the final comma at the end of this statement to really fix this :)

File: docs/content/docs/dev/table/sql/create.md [hunk 1]
```diff
@@ -316,7 +316,7 @@ CREATE TABLE MyTable (
   `user_id` BIGINT,
   `price` DOUBLE,
   `quantity` DOUBLE,
-  `cost` AS price * quanitity,  -- evaluate expression and supply the result to queries
+  `cost` AS price * quantity,  -- evaluate expression and supply the result to queries
```
[the remark this answers is not in the document]
Comment (BoYiZhang) [reply docs/content/docs/dev/table/sql/create.md]: Thank you very much. It has been corrected

File: docs/content.zh/docs/dev/table/sql/create.md [hunk 2]
```diff
@@ -317,7 +317,7 @@ CREATE TABLE MyTable (
   `user_id` BIGINT,
   `price` DOUBLE,
   `quantity` DOUBLE,
-  `cost` AS price * quanitity,  -- evaluate expression and supply the result to queries
+  `cost` AS price * quantity,  -- evaluate expression and supply the result to queries
```
[the remark this answers is not in the document]
Comment (BoYiZhang) [reply docs/content.zh/docs/dev/table/sql/create.md]: Thank you very much. It has been corrected

**Review by MartijnVisser — APPROVED** [review]
LGTM, thanks @BoYiZhang

**Pull request merged**